docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -162,6 +163,95 @@ expected results. Instead, exclude CUB library implementation source code from
 your migration by adding ``--in-root-exclude=<path to CUB library source code>``
 to your migration command.
 
+How do I fix the hang issue of sycl code caused by work group level synchronization like group barrier used in conditional statement?

```

Comment:
How do I fix the issue of SYCL* code hanging due to work group level synchronization, such as a group barrier used in a conditional statement?

Reply:
same of the above